guidelines/add-new-component-docs.md

```diff
@@ -2,30 +2,29 @@
 
 When you add a new component to the `kyma/docs` folder, you must ensure that the content also shows on the `https://kyma-project.io/` website and in the Console UI. To do so, update the following files:
 
-- [`resources/core/charts/docs/charts/documentation/templates/docs-job.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/docs/charts/documentation/templates/docs-job.yaml)
-- [`docs/manifest.yaml`](https://github.com/kyma-project/kyma/blob/master/docs/manifest.yaml)
-- [`kyma/docs/docs-build.yaml`](https://github.com/kyma-project/kyma/blob/master/docs/docs-build.yaml)
+- [`docs-job.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/docs/charts/documentation/templates/docs-job.yaml) - defines Docker images with documentation which are run to upload this documentation to Minio.
+- [`manifest.yaml`](https://github.com/kyma-project/kyma/blob/master/docs/manifest.yaml) - specifies the order of documentation topics on the website. If you do not include the component in this file, it doesn't show on the UI.
+- [`docs-build.yaml`](https://github.com/kyma-project/kyma/blob/master/docs/docs-build.yaml) - specifies all documentation topics needed for building Docker images. Every item must have the `name` field specified, which represents the name of the Docker image for a topic. The final image name is followed by a `-docs` suffix. If the name of the new component directory matches the name of the image, you don’t have to specify it. Otherwise, add an additional `directory` property.
```

Comment:
```suggestion
- [`docs-build.yaml`](https://github.com/kyma-project/kyma/blob/master/docs/docs-build.yaml) - specifies all documentation topics needed for building Docker images. Every item must have the **name** field specified, which represents the name of the Docker image for a topic. The final image name is followed by a `-docs` suffix. If the name of the new component directory matches the name of the image, you don’t have to specify it. Otherwise, add an additional **directory** property.
```